tools/metilene/tool_dependencies.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="perl" version="5.18.1">
+        <repository name="package_perl_5_18" owner="iuc" prior_installation_required="False" />
+    </package>
+    <package name="R" version="3.2.1">
+        <repository name="package_r_3_2_1" owner="iuc" />
+    </package>
+    <package name="bedtools" version="2.24">
+        <repository name="package_bedtools_2_24" owner="iuc" />
+    </package>
+    <package name="gnu_coreutils" version="8.22">
+        <repository name="package_gnu_coreutils_8_22" owner="iuc" />
+    </package>
+
```

Comment:
Any reason to not depend on https://testtoolshed.g2.bx.psu.edu/view/iuc/package_r_ggplot2_0_9_3/ade406d47752